roles/initramfs/README.md

```diff
@@ -6,17 +6,16 @@ The role is designed to be internal for this collection and support the automati
 
 ## Contents
 
-To allow fast fail, the role provides a [`preflight.yml`](./tasks/preflight.yml) tasks file to be used at the start of the playbook.
-Please note that the [`main`](./tasks/main.yml) task file will not run the preflight checks
+To allow fast fail, the role provides a [`preflight.yml`](./tasks/preflight.yml) tasks file to that should be included early in the play that ultimately includes the [`main`](./tasks/main.yml) role that actually reboots the host. Refer the usage section below for example.
```

Comment:
I think the `to` in `tasks file to that` should be removed

Reply:
Fixed it, thanks!